data_dictionary.md

```diff
@@ -0,0 +1,37 @@
+
+# Example Activity Stream Log
+
+{"session_duration":1635,"locale":"en-US","ip":"10.192.171.13","date":"2016-03-07",
+  "unload_reason":"close","client_id":"374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e","max_scroll_depth":145,"addon_version":"1.0.0",
+  "total_history_size":9,"ver":"3","ua":"python-requests\/2.9.1","click_position":"100x100","source":"top_sites",
+  "timestamp":1457396660000,"action":"activity_stream","tab_id":3,"load_reason":"restore","total_bookmarks":19}
```

Comment:
nit: wrap the JSON blob in the olde triple backtick so it renders as a code block.